### PR TITLE
[FLINK-31675][Connectors/AWS] Update AWS SDKv2 to v2.20.32 for 4.x

### PR DESCRIPTION
**Pull request by antovespoli**
## Purpose of the change

AWS SDK v2 from 2.19.14 to 2.20.32
Reorder NOTICE files for consistency

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Significant changes
- [ x ] Dependencies have been added or upgraded
- [ ] Public API has been changed (Public API is any class annotated with `@Public(Evolving)`)
- [ ] Serializers have been changed
- [ ] New feature has been introduced
  - If yes, how is this documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Review by hlteoh37 — APPROVED** [review]
LGTM ty

**Comment by hlteoh37** [comment]
can we squash the commits into 1 please?

File: flink-connector-kinesis/src/main/resources/META-INF/NOTICE [hunk 1]
```diff
@@ -8,27 +8,27 @@ This project bundles the following dependencies under the Apache Software Licens
 
 - software.amazon.ion:ion-java:1.0.2
 - software.amazon.eventstream:eventstream:1.0.1
```
Comment (hlteoh37) [inline flink-connector-kinesis/src/main/resources/META-INF/NOTICE:10]: Verified these are all correct based on build checks

```
[INFO] --- maven-shade-plugin:3.1.1:shade (shade-flink) @ flink-connector-kinesis ---
[INFO] Including com.amazonaws:aws-java-sdk-kinesis:jar:1.12.276 in the shaded jar.
[INFO] Including com.amazonaws:aws-java-sdk-core:jar:1.12.276 in the shaded jar.
[INFO] Including software.amazon.ion:ion-java:jar:1.0.2 in the shaded jar.
[INFO] Including com.amazonaws:jmespath-java:jar:1.12.276 in the shaded jar.
[INFO] Including com.amazonaws:aws-java-sdk-sts:jar:1.12.276 in the shaded jar.
[INFO] Including com.amazonaws:aws-java-sdk-kms:jar:1.12.276 in the shaded jar.
[INFO] Including com.amazonaws:aws-java-sdk-s3:jar:1.12.276 in the shaded jar.
[INFO] Including com.amazonaws:aws-java-sdk-dynamodb:jar:1.12.276 in the shaded jar.
[INFO] Including com.amazonaws:aws-java-sdk-cloudwatch:jar:1.12.276 in the shaded jar.
[INFO] Including com.amazonaws:amazon-kinesis-producer:jar:0.14.1 in the shaded jar.
[INFO] Excluding org.slf4j:slf4j-api:jar:1.7.36 from the shaded jar.
[INFO] Including com.google.protobuf:protobuf-java:jar:3.21.7 in the shaded jar.
[INFO] Excluding jakarta.xml.bind:jakarta.xml.bind-api:jar:2.3.2 from the shaded jar.
[INFO] Excluding jakarta.activation:jakarta.activation-api:jar:1.2.1 from the shaded jar.
[INFO] Including com.amazonaws:amazon-kinesis-client:jar:1.14.8 in the shaded jar.
[INFO] Including com.fasterxml.jackson.core:jackson-core:jar:2.13.4 in the shaded jar.
[INFO] Including com.amazonaws:dynamodb-streams-kinesis-adapter:jar:1.5.3 in the shaded jar.
[INFO] Including com.fasterxml.jackson.core:jackson-databind:jar:2.13.4.2 in the shaded jar.
[INFO] Including com.fasterxml.jackson.core:jackson-annotations:jar:2.13.4 in the shaded jar.
[INFO] Excluding joda-time:joda-time:jar:2.8.1 from the shaded jar.
[INFO] Excluding commons-codec:commons-codec:jar:1.15 from the shaded jar.
[INFO] Excluding commons-io:commons-io:jar:2.11.0 from the shaded jar.
[INFO] Excluding commons-lang:commons-lang:jar:2.6 from the shaded jar.
[INFO] Excluding commons-logging:commons-logging:jar:1.1.3 from the shaded jar.
[INFO] Excluding org.apache.commons:commons-lang3:jar:3.12.0 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-connector-base:jar:1.17.0 from the shaded jar.
[INFO] Excluding com.google.guava:guava:jar:29.0-jre from the shaded jar.
[INFO] Excluding com.google.guava:failureaccess:jar:1.0.1 from the shaded jar.
[INFO] Excluding com.google.guava:listenablefuture:jar:9999.0-empty-to-avoid-conflict-with-guava from the shaded jar.
[INFO] Excluding com.google.code.findbugs:jsr305:jar:1.3.9 from the shaded jar.
[INFO] Excluding org.checkerframework:checker-qual:jar:2.11.1 from the shaded jar.
[INFO] Excluding com.google.errorprone:error_prone_annotations:jar:2.3.4 from the shaded jar.
[INFO] Excluding com.google.j2objc:j2objc-annotations:jar:1.3 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-core:jar:1.17.0 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-shaded-guava:jar:30.1.1-jre-16.1 from the shaded jar.
[INFO] Including org.apache.flink:flink-connector-aws-kinesis-streams:jar:4.2-SNAPSHOT in the shaded jar.
[INFO] Excluding org.apache.flink:flink-table-common:jar:1.17.0 from the shaded jar.
[INFO] Including org.apache.flink:flink-connector-aws-base:jar:4.2-SNAPSHOT in the shaded jar.
[INFO] Including software.amazon.awssdk:apache-client:jar:2.20.32 in the shaded jar.
[INFO] Excluding org.apache.flink:flink-annotations:jar:1.17.0 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-metrics-core:jar:1.17.0 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-shaded-asm-9:jar:9.3-16.1 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-shaded-jackson:jar:2.13.4-16.1 from the shaded jar.
[INFO] Excluding org.apache.commons:commons-text:jar:1.10.0 from the shaded jar.
[INFO] Excluding com.esotericsoftware.kryo:kryo:jar:2.24.0 from the shaded jar.
[INFO] Excluding com.esotericsoftware.minlog:minlog:jar:1.2 from the shaded jar.
[INFO] Excluding commons-collections:commons-collections:jar:3.2.2 from the shaded jar.
[INFO] Excluding org.apache.commons:commons-compress:jar:1.21 from the shaded jar.
[INFO] Excluding com.ibm.icu:icu4j:jar:67.1 from the shaded jar.
[INFO] Including software.amazon.awssdk:kinesis:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-cbor-protocol:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:third-party-jackson-dataformat-cbor:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:third-party-jackson-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-json-protocol:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:protocol-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:sdk-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:auth:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.eventstream:eventstream:jar:1.0.1 in the shaded jar.
[INFO] Including software.amazon.awssdk:http-client-spi:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:regions:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:annotations:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:utils:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:metrics-spi:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:json-utils:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:endpoints-spi:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:netty-nio-client:jar:2.20.32 in the shaded jar.
[INFO] Including io.netty:netty-codec-http:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-codec-http2:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-codec:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-transport:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-resolver:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-common:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-buffer:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-handler:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-transport-native-unix-common:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-transport-classes-epoll:jar:4.1.86.Final in the shaded jar.
[INFO] Including org.reactivestreams:reactive-streams:jar:1.0.3 in the shaded jar.
[INFO] Including software.amazon.awssdk:sts:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-query-protocol:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:profiles:jar:2.20.32 in the shaded jar.
[INFO] Including org.apache.httpcomponents:httpclient:jar:4.5.13 in the shaded jar.
[INFO] Including org.apache.httpcomponents:httpcore:jar:4.4.14 in the shaded jar.
[INFO] Including com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:jar:2.13.4 in the shaded jar.
[INFO] Excluding org.apache.flink:flink-shaded-force-shading:jar:16.0 from the shaded jar.
[INFO] Excluding org.objenesis:objenesis:jar:2.1 from the shaded jar.
[INFO] No artifact matching filter io.netty:netty
```

File: flink-sql-connector-aws-kinesis-streams/src/main/resources/META-INF/NOTICE [hunk 2]
```diff
@@ -7,27 +7,27 @@ The Apache Software Foundation (http://www.apache.org/).
...
-- software.amazon.awssdk:http-client-spi:2.19.14
-- software.amazon.awssdk:endpoints-spi:2.19.14
-- software.amazon.awssdk:aws-query-protocol:2.19.14
-- software.amazon.awssdk:aws-json-protocol:2.19.14
-- software.amazon.awssdk:aws-core:2.19.14
-- software.amazon.awssdk:aws-cbor-protocol:2.19.14
-- software.amazon.awssdk:auth:2.19.14
-- software.amazon.awssdk:apache-client:2.19.14
-- software.amazon.awssdk:annotations:2.19.14
+- software.amazon.awssdk:utils:2.20.32
```
Comment (hlteoh37) [inline flink-sql-connector-aws-kinesis-streams/src/main/resources/META-INF/NOTICE:10]: Verified correct based on build output

```
[INFO] --- maven-shade-plugin:3.1.1:shade (shade-flink) @ flink-sql-connector-aws-kinesis-streams ---
[INFO] Including org.apache.flink:flink-connector-aws-kinesis-streams:jar:4.2-SNAPSHOT in the shaded jar.
[INFO] Including org.apache.flink:flink-connector-base:jar:1.17.0 in the shaded jar.
[INFO] Including org.apache.flink:flink-connector-aws-base:jar:4.2-SNAPSHOT in the shaded jar.
[INFO] Including software.amazon.awssdk:netty-nio-client:jar:2.20.32 in the shaded jar.
[INFO] Including io.netty:netty-codec-http:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-codec-http2:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-codec:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-transport:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-resolver:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-common:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-buffer:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-handler:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-transport-native-unix-common:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-transport-classes-epoll:jar:4.1.86.Final in the shaded jar.
[INFO] Including org.reactivestreams:reactive-streams:jar:1.0.3 in the shaded jar.
[INFO] Including software.amazon.awssdk:apache-client:jar:2.20.32 in the shaded jar.
[INFO] Including org.apache.httpcomponents:httpclient:jar:4.5.13 in the shaded jar.
[INFO] Including commons-logging:commons-logging:jar:1.1.3 in the shaded jar.
[INFO] Including org.apache.httpcomponents:httpcore:jar:4.4.14 in the shaded jar.
[INFO] Excluding commons-codec:commons-codec:jar:1.15 from the shaded jar.
[INFO] Including software.amazon.awssdk:sts:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-query-protocol:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:profiles:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:kinesis:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-cbor-protocol:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:third-party-jackson-dataformat-cbor:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:third-party-jackson-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-json-protocol:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:protocol-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:sdk-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:auth:jar:2.20.32 in the shaded jar.
[INFO] Excluding software.amazon.eventstream:eventstream:jar:1.0.1 from the shaded jar.
[INFO] Including software.amazon.awssdk:http-client-spi:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:regions:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:annotations:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:utils:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:metrics-spi:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:json-utils:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:endpoints-spi:jar:2.20.32 in the shaded jar.
[INFO] Excluding org.apache.flink:flink-table-common:jar:1.17.0 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-core:jar:1.17.0 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-shaded-asm-9:jar:9.3-16.1 from the shaded jar.
[INFO] Excluding com.ibm.icu:icu4j:jar:67.1 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-shaded-force-shading:jar:16.0 from the shaded jar.
[INFO] Excluding org.apache.commons:commons-compress:jar:1.21 from the shaded jar.
[INFO] Excluding org.objenesis:objenesis:jar:2.1 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-shaded-guava:jar:30.1.1-jre-16.1 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-shaded-jackson:jar:2.13.4-16.1 from the shaded jar.
[INFO] Excluding org.apache.commons:commons-lang3:jar:3.12.0 from the shaded jar.
[INFO] Excluding org.apache.commons:commons-text:jar:1.10.0 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-annotations:jar:1.17.0 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-metrics-core:jar:1.17.0 from the shaded jar.
[INFO] Excluding com.esotericsoftware.kryo:kryo:jar:2.24.0 from the shaded jar.
[INFO] Excluding com.esotericsoftware.minlog:minlog:jar:1.2 from the shaded jar.
[INFO] Excluding commons-collections:commons-collections:jar:3.2.2 from the shaded jar.
[INFO] Excluding org.slf4j:slf4j-api:jar:1.7.36 from the shaded jar.
[INFO] Excluding com.google.code.findbugs:jsr305:jar:1.3.9 from the shaded jar.
```

File: flink-sql-connector-aws-kinesis-firehose/src/main/resources/META-INF/NOTICE [hunk 3]
```diff
@@ -7,25 +7,25 @@ The Apache Software Foundation (http://www.apache.org/).
...
-- software.amazon.awssdk:http-client-spi:2.19.14
-- software.amazon.awssdk:firehose:2.19.14
-- software.amazon.awssdk:endpoints-spi:2.19.14
-- software.amazon.awssdk:aws-query-protocol:2.19.14
-- software.amazon.awssdk:aws-json-protocol:2.19.14
-- software.amazon.awssdk:aws-core:2.19.14
-- software.amazon.awssdk:auth:2.19.14
-- software.amazon.awssdk:apache-client:2.19.14
-- software.amazon.awssdk:annotations:2.19.14
+- software.amazon.awssdk:utils:2.20.32
```
Comment (hlteoh37) [inline flink-sql-connector-aws-kinesis-firehose/src/main/resources/META-INF/NOTICE:10]: Verified ok using build output

```

[INFO] --- maven-shade-plugin:3.1.1:shade (shade-flink) @ flink-sql-connector-aws-kinesis-firehose ---
[INFO] Including org.apache.flink:flink-connector-aws-kinesis-firehose:jar:4.2-SNAPSHOT in the shaded jar.
[INFO] Including org.apache.flink:flink-connector-base:jar:1.17.0 in the shaded jar.
[INFO] Including org.apache.flink:flink-connector-aws-base:jar:4.2-SNAPSHOT in the shaded jar.
[INFO] Including software.amazon.awssdk:apache-client:jar:2.20.32 in the shaded jar.
[INFO] Including org.apache.httpcomponents:httpclient:jar:4.5.13 in the shaded jar.
[INFO] Including commons-logging:commons-logging:jar:1.1.3 in the shaded jar.
[INFO] Including org.apache.httpcomponents:httpcore:jar:4.4.14 in the shaded jar.
[INFO] Excluding commons-codec:commons-codec:jar:1.15 from the shaded jar.
[INFO] Including software.amazon.awssdk:sts:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-query-protocol:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:profiles:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:firehose:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-json-protocol:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:third-party-jackson-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:protocol-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:sdk-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:auth:jar:2.20.32 in the shaded jar.
[INFO] Excluding software.amazon.eventstream:eventstream:jar:1.0.1 from the shaded jar.
[INFO] Including software.amazon.awssdk:http-client-spi:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:regions:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:annotations:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:utils:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:metrics-spi:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:json-utils:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:endpoints-spi:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:netty-nio-client:jar:2.20.32 in the shaded jar.
[INFO] Including io.netty:netty-codec-http:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-codec-http2:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-codec:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-transport:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-resolver:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-common:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-buffer:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-handler:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-transport-native-unix-common:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-transport-classes-epoll:jar:4.1.86.Final in the shaded jar.
[INFO] Including org.reactivestreams:reactive-streams:jar:1.0.3 in the shaded jar.
[INFO] Excluding org.apache.flink:flink-table-common:jar:1.17.0 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-core:jar:1.17.0 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-shaded-asm-9:jar:9.3-16.1 from the shaded jar.
[INFO] Excluding com.ibm.icu:icu4j:jar:67.1 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-shaded-force-shading:jar:16.0 from the shaded jar.
[INFO] Excluding org.apache.commons:commons-compress:jar:1.21 from the shaded jar.
[INFO] Excluding org.objenesis:objenesis:jar:2.1 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-shaded-guava:jar:30.1.1-jre-16.1 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-shaded-jackson:jar:2.13.4-16.1 from the shaded jar.
[INFO] Excluding org.apache.commons:commons-lang3:jar:3.12.0 from the shaded jar.
[INFO] Excluding org.apache.commons:commons-text:jar:1.10.0 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-annotations:jar:1.17.0 from the shaded jar.
[INFO] Excluding org.apache.flink:flink-metrics-core:jar:1.17.0 from the shaded jar.
[INFO] Excluding com.esotericsoftware.kryo:kryo:jar:2.24.0 from the shaded jar.
[INFO] Excluding com.esotericsoftware.minlog:minlog:jar:1.2 from the shaded jar.
[INFO] Excluding commons-collections:commons-collections:jar:3.2.2 from the shaded jar.
[INFO] Excluding org.slf4j:slf4j-api:jar:1.7.36 from the shaded jar.
[INFO] Excluding com.google.code.findbugs:jsr305:jar:1.3.9 from the shaded jar.
```

File: flink-sql-connector-dynamodb/src/main/resources/META-INF/NOTICE [hunk 4]
```diff
@@ -7,26 +7,26 @@ The Apache Software Foundation (http://www.apache.org/).
...
-- software.amazon.awssdk:endpoints-spi:2.19.14
-- software.amazon.awssdk:dynamodb:2.19.14
-- software.amazon.awssdk:dynamodb-enhanced:2.19.14
-- software.amazon.awssdk:aws-query-protocol:2.19.14
-- software.amazon.awssdk:aws-json-protocol:2.19.14
-- software.amazon.awssdk:aws-core:2.19.14
-- software.amazon.awssdk:auth:2.19.14
-- software.amazon.awssdk:apache-client:2.19.14
-- software.amazon.awssdk:annotations:2.19.14
+- software.amazon.awssdk:utils:2.20.32
```
Comment (hlteoh37) [inline flink-sql-connector-dynamodb/src/main/resources/META-INF/NOTICE:10]: Verfieid ok from build output

```
INFO] --- maven-shade-plugin:3.1.1:shade (shade-flink) @ flink-sql-connector-dynamodb ---
[INFO] Including org.apache.flink:flink-connector-dynamodb:jar:4.2-SNAPSHOT in the shaded jar.
[INFO] Including org.apache.flink:flink-connector-base:jar:1.17.0 in the shaded jar.
[INFO] Including org.apache.flink:flink-connector-aws-base:jar:1.17.0 in the shaded jar.
[INFO] Including software.amazon.awssdk:netty-nio-client:jar:2.20.32 in the shaded jar.
[INFO] Including io.netty:netty-codec-http:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-codec-http2:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-codec:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-transport:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-resolver:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-common:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-buffer:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-handler:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-transport-native-unix-common:jar:4.1.86.Final in the shaded jar.
[INFO] Including io.netty:netty-transport-classes-epoll:jar:4.1.86.Final in the shaded jar.
[INFO] Including org.reactivestreams:reactive-streams:jar:1.0.3 in the shaded jar.
[INFO] Including software.amazon.awssdk:sts:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-query-protocol:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:dynamodb:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-json-protocol:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:third-party-jackson-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:protocol-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:profiles:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:sdk-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:auth:jar:2.20.32 in the shaded jar.
[INFO] Excluding software.amazon.eventstream:eventstream:jar:1.0.1 from the shaded jar.
[INFO] Including software.amazon.awssdk:http-client-spi:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:regions:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:annotations:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:utils:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:aws-core:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:metrics-spi:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:json-utils:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:endpoints-spi:jar:2.20.32 in the shaded jar.
[INFO] Including software.amazon.awssdk:apache-client:jar:2.20.32 in the shaded jar.
[INFO] Including org.apache.httpcomponents:httpclient:jar:4.5.13 in the shaded jar.
[INFO] Including commons-logging:commons-logging:jar:1.1.3 in the shaded jar.
[INFO] Including org.apache.httpcomponents:httpcore:jar:4.4.14 in the shaded jar.
[INFO] Including commons-codec:commons-codec:jar:1.15 in the shaded jar.
[INFO] Including software.amazon.awssdk:dynamodb-enhanced:jar:2.20.32 in the shaded jar.
[INFO] Excluding org.apache.flink:flink-shaded-force-shading:jar:16.0 from the shaded jar.
[INFO] Excluding org.slf4j:slf4j-api:jar:1.7.36 from the shaded jar.
````

**Review by hlteoh37 — APPROVED** [review]
LGTM

**Review by dannycranmer — APPROVED** [review]

**Pull request merged**